openshift/ci-operator/build-image/cr.yaml

```diff
@@ -14,10 +14,10 @@ spec:
       redundancyPolicy: "ZeroRedundancy"
       resources:
         limits:
-          memory: 3Gi
+          memory: 2Gi
         requests:
-          cpu: 500m
-          memory: 3Gi
+          cpu: 100m
+          memory: 2Gi
```

Comment:
Can you comment on the reason for these changes? Are they arbitrary?

Reply:
We have no specific needs for these values.  I lowered them to ensure we could land the ES nodes without having to bump the OCP sizes.